src/Features/Core/Portable/EditAndContinue/AbstractEditAndContinueAnalyzer.cs

```diff
@@ -421,6 +422,21 @@ protected virtual string GetSuspensionPointDisplayName(SyntaxNode node, EditKind
         /// </summary>
         internal abstract bool IsDeclarationWithInitializer(SyntaxNode declaration);
 
+        /// <summary>
+        /// Return true if the declaration is a parameter that is part of a records primary constructor.
+        /// </summary>
+        internal abstract bool IsRecordPrimaryConstructorParameter(SyntaxNode declaration);
+
+        /// <summary>
+        /// Return true if the declaration is a property that represents one of the parameters in a records primary constructor.
+        /// </summary>
+        internal abstract bool IsRecordPrimaryConstructorProperty(SyntaxNode declaration);
```

Comment:
>IsRecordPrimaryConstructorProperty [](start = 31, length = 34)

Consider renaming to `IsRecordPropertyMatchingPrimaryConstructorParameter`.

Reply:
Makes sense, renamed